libs/comparator/tables.js

```diff
@@ -127,6 +160,8 @@ const buildAppTablesMap = apps => {
 }
 
 module.exports = apps => {
+    enrichColumnsWithRelationIdentificationData(...apps)
```

Comment:
please notice the printDifferences method invocation two rows above.
This is the purpose of this module. to compare apps and print differences in human-readable format

Reply:
@vengrov the `buildAppTablesMap` which is exported from this module is reused in table sync logic.

If we want to be able to print `relatedIdentificationColumnName` difference between `schema` and `target app`, we need to resolve `relatedIdentificationColumnName` from `relatedIdentificationColumnId` for the target app.

And for updating the data, we need to resolve `relatedIdentificationColumnId` from `relatedIdentificationColumnName` for the schema.

My point is this module is used not only for printing but for synching as well, that's why I put this logic here. please let me know if it's still wrong.